duecredit/collector.py

```diff
@@ -19,10 +19,12 @@
 from .stub import InactiveDueCreditCollector
 from .io import TextOutput, PickleOutput
 from .utils import never_fail, borrowdoc, external_versions
+from collections import namedtuple
 
 import logging
 lgr = logging.getLogger('duecredit.collector')
 
+CitationKey = namedtuple('CitationKey', ['path', 'entry_key'])
```

Comment:
I think we should also bring version here as well.  originally I thought that we might just store multiple  versions within citation  but probably it would not be the kosher way -  we should resolve presence  ofmultiple  versions when preparing output.